src/ModelButtonImage/ModelButtonImage.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { useTheme } from "@mui/material/styles";
+
+// ModelButtonImage component is used to display the model icon in the ModelButton component.
+export default function ModelButtonImage({ src }) {
+  const theme = useTheme();
+
+  return (
+    <>
+      {src && src.length > 0 ? (
```

Comment:
what will happen in ModelButton if no src? This component should always expect src so that make the prop required